### DISCUSSION TITLE
安装了.NET运行时之后依然提示缺少运行时

### DISCUSSION THREAD

**Discussion by ghost** [HMBSbige/NatTypeTester#209] [Q&A] [answered]
我已经安装好了.NET 7.0.5运行时，但是依然提示缺少运行时。

![图1](https://i.im.ge/2023/05/06/UdihtT.Snipaste-2023-05-06-12-56-53.png)
![图2](https://i.im.ge/2023/05/06/Udi5FL.Snipaste-2023-05-06-12-57-39.png)

**Comment by HMBSbige** [comment]
因为是 6.0 的呢

**Comment by ZZZZzzzzac** [comment]
~~同样问题, 装了6.0~~
问题解决， 是要装上面的Desktop Runtime，（不知道下面的Runtime是否也需要）
![image](https://github.com/HMBSbige/NatTypeTester/assets/17446384/acf8d770-43dc-40c5-a6ac-2b0a2558553e)

**Reply by abbaccadd0** [reply, marked as answer]
.Net Desktop Runtime中包含.Net Runtime
因此不需要安装下面的（实际上是已经装完了
